lisa/core/node.py

```diff
@@ -217,7 +217,7 @@ def _execute(
         useBash: bool = False,
         noErrorLog: bool = False,
         noInfoLog: bool = False,
-        cwd: pathlib.Path = None,
+        cwd: pathlib.Path = pathlib.Path(),
```

Comment:
Is it good to have an empty one? The None can be ignored in underlying API, but I'm not sure about empty path.